WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift

```diff
@@ -6,22 +6,33 @@ class LoginEpilogueViewController: UIViewController {
     var dismissBlock: ((_ cancelled: Bool) -> Void)?
     @IBOutlet var buttonPanel: UIView?
     @IBOutlet var shadowView: UIView?
+    @IBOutlet var connectButton: UIButton?
```

Comment:
I still think "!" is the proper approach (I should blog my reasoning tbh) but I wouldn't be able to stand the inconsistency if I chose that option here. ;)